webserver/static/scripts/datasets/eval-jobs-viewer.js

```diff
@@ -61,14 +61,16 @@ var EvaluationJobsViewer = React.createClass({
 
         // Hash is used to store ID of the job that is currently viewed.
         window.addEventListener('hashchange', this.handleHashChange);
+
     },
     componentWillUnmount: function() {
         window.removeEventListener('hashchange', this.handleHashChange);
     },
     handleHashChange: function(e) {
         // Hash is used to store ID of the currently viewed job.
         if (this.state.jobs) {
-            var hash = window.location.hash.substr(1);
+            //var hash = window.location.hash.substr(1);
+            var hash = this.state.active_job_index;
```

Comment:
This variable name made sense when we were reading the url hash (the part after the #). Perhaps now we should call it `job_id`